src/commands/createWebApp/setDefaultRgAndPlanName.ts

```diff
@@ -91,12 +92,12 @@ async function promptPerformanceWarning(context: IActionContext, asp: AppService
         const input: MessageItem = await ext.ui.showWarningMessage(`The selected plan currently has ${numberOfSites} apps. Deploying more than ${maxNumberOfSites} apps may degrade the performance on the apps in the plan.`, { modal: true }, ...inputs);
 
         if (input === AppServiceDialogResponses.turnOnAdvancedCreation) {
-            await workspaceConfig.update('advancedCreation', true, ConfigurationTarget.Global);
+            await updateGlobalSetting(configurationSettings.advancedCreation, true);
             context.telemetry.properties.cancelStep = AppServiceDialogResponses.turnOnAdvancedCreation.title;
             throw new UserCancelledError();
         } else if (input === DialogResponses.dontWarnAgain) {
             context.telemetry.properties.turnOffPerfWarning = 'true';
-            workspaceConfig.update(showPlanPerformanceWarningSetting, false, ConfigurationTarget.Global);
+            await updateGlobalSetting(showPlanPerformanceWarningSetting, false);
```

Comment:
Any chance you could move these changes to a separate PR? The addition of "../../vsCodeConfig/settings"; affects a lot of files so I find myself re-reviewing it over and over, but it's pretty simple stuff by itself and could've been merged already